styles.css

```diff
@@ -30,7 +48,266 @@
 /* ------------------------------- 4. Areas ---------------------------------------- */
 
 /* --------------- 4.1 Header ------------------------- */
+.navbar-outer-container {
+    display: flex;
+    background: rgb(72,69,228);
+    background: linear-gradient(168deg, rgba(72,69,228,1) 0%, rgba(87,34,205,1) 35%);
+}
+
+
+.navbar-items-inner-container {
+    background-color: #5722CD;
+    font-family: "Roboto", sans-serif;
+    text-decoration-color: white;
+    display: flex;
+    padding: 25px;
+    justify-content: end;
+    list-style: none;
+    gap: 100px;
+    color: white;
+
+}
+
+.navbar-items-inner-container a:link {
+    color: white;
+}
+.navbar-items-inner-container a:visited {
+    color: white;
+}
 
 /* --------------- 4.2 Introduction ------------------- */
+.intro-outer-container-container {
+    display: flex;
+    flex-direction: column;
+    background-color: #5722CD;
+    margin-bottom: 25px;
+}
+
+.intro-h1-inner-container {
+    display: flex;
+
+    text-align: center;
+    color: white;
+    margin-top: 100px;
+    margin-right: 400px;
+    margin-left: 400px;
+    margin-bottom: 25px;
+    font-size: 40px;
+}
+
+.intro-inner-container {
+    display: flex;
+    flex-direction: column;
+    align-content: center;
+    text-align: center;
+    color: white;
+    font-family: "Roboto", sans-serif;
+    margin-right: 400px;
+    margin-left: 400px;
+    margin-bottom: 25px;
+    font-size: 15px;
+}
+
+.button {
```

Comment:
Het zou nog mooier zijn om bovenaan het bestand wat generieke styling voor alle buttons op de pagina te declareren en alles dat specifiek is voor deze header-button dan hier te overschrijven. Scheelt ook weer dubbele code